DetectorTomography.py

```diff
@@ -44,137 +44,46 @@ def gtc_matrix_product_counting_function(arguments: list):
     return result
 
 
-class DetectorTomographyFitter:
-    """
-        This class is meant to resemble qiskit's state tomography and process tomography fitters and to calculate the
-        maximum likelihood povm estimator describing a detector basing on QDT job results and used probe states.
-    """
+class QDTCalibrationSetup:
 
```

Comment:
This class needs a general description (but I guess that's obvious) :-) 
As of now, I do not see here how to use non-qiskit results. It would require adding something like "__get_frequencies_array_from_results" but for either list of lists (possibly vectors), either for a list of dicts. This is quite easy to code so shouldn't be a problem :-)

Reply:
That's what general constructor is for. Given that you provide frequencies_array, qubits_number and some other data (they can be from qiskit, riggeti or obtained otherwise) you are able to create setup object using default constructor. 

As for description -- you're right, I'll add it. Thanks.